docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,657 @@ license: |
+      <td>any</td>
+      <td>Collects and returns a list of non-unique elements. The function is non-deterministic because the order of collected results depends on the order of the rows which may be non-deterministic after a shuffle.</td>
+    </tr>       
+    <tr>
+      <td><b>collect_set</b>(<i>expression</i>)</td>
+      <td>any</td>
+      <td>Collects and returns a set of unique elements. The function is non-deterministic because the order of collected results depends on the order of the rows which may be non-deterministic after a shuffle.</td>
+    </tr>
+    <tr>
+      <td><b>corr</b>(<i>expression1, expression2</i>)</td>
+      <td>(double, double)</td>
+      <td>Returns Pearson coefficient of correlation between a set of number pairs.</td>
+    </tr>
+    <tr>
+      <td><b>count</b>([<b>DISTINCT</b>] <i>*</i>)</td>
+      <td>none</td>
+      <td>If specified <code>DISTINCT</code>, returns the total number of retrieved rows are unique and not null; Otherwise, returns the total number of retrieved rows, including rows containing null.</td>
+    </tr>
+    <tr>
+      <td><b>count</b>([<b>DISTINCT</b>] <i>expression1[, expression2</i>])</td>
+      <td>(any, any)</td>
+      <td>If specified <code>DISTINCT</code>, returns the number of rows for which the supplied expression(s) are unique and not null; Otherwise, returns the number of rows for which the supplied expression(s) are all not null.</td>
+    </tr>
+    <tr>
+      <td><b>count_if</b>(<i>predicate</i>)</td>
+      <td>expression that will be used for aggregation calculation</td>
+      <td>Returns the count number from the predicate evaluate to `TRUE` values.</td>
+    </tr> 
+    <tr>
+      <td><b>count_min_sketch</b>(<i>expression, eps, confidence, seed</i>)</td>
+      <td>(byte, short, int, long, string or binary, double,  double, integer)</td>
+      <td>`eps` and `confidence` are the double values between 0.0 and 1.0, `seed` is a positive integer. Returns a count-min sketch of a expression with the given `esp`, `confidence` and `seed`. The result is an array of bytes, which can be deserialized to a `CountMinSketch` before usage. Count-min sketch is a probabilistic data structure used for cardinality estimation using sub-linear space.</td>
+    </tr>
+    <tr>
+      <td><b>covar_pop</b>(<i>expression1, expression2</i>)</td>
+      <td>(double, double)</td>
+      <td>Returns the population covariance of a set of number pairs.</td>
+    </tr> 
+    <tr>
+      <td><b>covar_samp</b>(<i>expression1, expression2</i>)</td>
+      <td>(double, double)</td>
+      <td>Returns the sample covariance of a set of number pairs.</td>
+    </tr>  
+    <tr>
+      <td><b>{first | first_value}</b>(<i>expression[, isIgnoreNull]</i>)</td>
+      <td>(any, boolean)</td>
+      <td>Returns the first value of expression for a group of rows. If <code>isIgnoreNull</code> is true, returns only non-null values, default is false. This function is non-deterministic.</td>
```

Comment:
`<code>isIgnoreNull</code>` -> `isIgnoreNull`?  Should we use \` or \<code\> for arguments?

Reply:
I think use `, seems we don't use `isIgnoreNull` in spark code. done